### PR TITLE
fix: UPM no longer expects exactly 1 artifact

### PR DESCRIPTION
**Pull request by bitsandfoxes**
The requirements for releasing the Unity SDK changed and with it the artifacts we upload

- Previously:
   Exactly one artifact `package-release.zip` that contains the UPM package.
- Now (with native support):
   Also contains symbols for the Symbols Collector

So we just need to pick that one `package-release.zip` from all the artifacts and we're good to go!

### CODE REVIEW

**Review by HazAT — APPROVED** [review]

File: src/targets/upm.ts [hunk 1]
```diff
@@ -30,6 +30,8 @@ export class UpmTarget extends BaseTarget {
   public readonly github: Octokit;
   /** Internal GitHub Target */
   private readonly githubTarget: GithubTarget;
+  /**  */
+  public readonly packageName: string = 'package-release.zip';
```
Comment (BYK) [inline src/targets/upm.ts:34]: Shouldn't this be a constant?

**Comment by BYK** [comment]
GitHub actions are broken so waiting till the builds pass.

**Review by BYK — APPROVED** [review]

File: src/targets/upm.ts (hunk 1 above)
Comment (bitsandfoxes) [reply src/targets/upm.ts]: It should! Changed it.

File: src/targets/upm.ts [hunk 2]
```diff
@@ -72,16 +75,17 @@ export class UpmTarget extends BaseTarget {
       reportError('Cannot publish UPM: No release artifact found.');
       return;
     }
-    if (packageFiles.length > 1) {
+
+    const packageFile = packageFiles.find(
+      packageFile => packageFile.filename === ARTIFACT_NAME
```
Comment (BYK) [inline src/targets/upm.ts:80]:
```suggestion
      ({ filename }) => filename === ARTIFACT_NAME
```

**Review by BYK — APPROVED** [review]

**Pull request merged**